### ISSUE TITLE
Update Pattern reconciliation monitor dashboard after signing/unsigning

### ISSUE THREAD

**Issue by LCCarmody** [obophenotype/pattern-review#5]
It would be helpful if the Pattern reconciliation monitor dashboard updated in someway to say that you have changed signing status of a pattern. 

Preferred change: Update number of signers, update 'done' category, and change color (if necessary) to completely update the whole list of patterns so you can easily see what has been changed.

Acceptable: A check mark or highlight in some way to signify that a pattern has just been changed--maybe with a message saying your changes have been recorded.

**Comment by LCCarmody** [comment]
Fyi, stopping the app and restarting did not appear to change anything. Will try again tomorrow.

**Comment by matentzn** [comment]
Yes, these changes only show up after the pull requests are merged in.
Thanks for the suggestions.
